.bazelrc

```
@@ -160,6 +160,7 @@ build:libc++ --define force_libcpp=enabled
 
 # Optimize build for binary size reduction.
 build:sizeopt -c opt --copt -Os --linkopt=-Wl,--gc-sections
+build:macossizeopt -c opt --copt -Os
```

This isn't macOS-specific, it's non-ELF, which is all apple platforms and Windows.

```suggestion
build:nonelfsizeopt -c opt --copt -Os
```

Don't love that name either. Naming wise it's definitely easier to flip the defaults with `sizeopt` and `gcsections`,